README.md

```diff
@@ -9,12 +9,17 @@ now only need to be `pub(crate)`, `pub(super)`, or simply private.
 
 `depub` minimises the visibility of such items in files passed to it, using a
 user-specified command (e.g. `cargo check`) as an oracle to tell if its
-reduction of an item's visibility is valid or not. In essence, `depub` does a
-string search for `pub`, replaces it with `pub crate` and sees if a test
-command still succeeds. If it does, it keeps that visibility, otherwise it
-replaces with the original and tries the next item. Note that `depub` is
-inherently destructive: it overwrites files as it operates, so do not run it on
-source code that you do not want altered!
+reduction of an item's visibility is valid or not. Note that `depub` is
+entirely guided by the oracle command: if the code it compiles happens not to
+use part of an intentionally public interface, then `depub` is likely to
+suggest reducing its visibility even though that's what you want. The broader
```

Comment:
> that's **not** what you want

I think?

Reply:
Fixed in https://github.com/softdevteam/depub/pull/1/commits/7f988339e75ec34976c26f59d2bdc2d83ddde97f.